python/paddle/fluid/contrib/slim/tests/qat_int8_nlp_comparison.py

```diff
@@ -251,6 +256,7 @@ def test_graph_transformation(self):
 
         _logger.info('QAT FP32 & INT8 prediction run.')
```

Comment:
Here it is not necessarily QAT FP32, it can also be FP32 original. Maybe we can set it a variable and pass into this log.